autovizwidget/setup.py

```diff
@@ -58,8 +58,8 @@ def version(path):
         'Programming Language :: Python :: 3.4'],
       install_requires=[
           'plotly>=1.10.0,<1.11.0',
-          'ipywidgets>5.0.0,<7.0',
+          'ipywidgets>=7.0.0a4,<8.0',
```

Comment:
Sorry, but let's not bump to any alpha version whatsoever. It should be `>=6.1.0,<7.0`